docs/api/en/core/BufferGeometry.html

```diff
@@ -236,7 +236,7 @@ <h3>[method:BufferGeometry fromGeometry]( [param:Geometry] )</h3>
 		<h3>[method:BufferAttribute getAttribute]( [param:String name] )</h3>
 		<p>Returns the [page:BufferAttribute attribute] with the specified name.</p>
 
-		<h3>[method:BufferAttribute getIndex] ()</h3>
+		<h3>[method:BufferAttribute getIndex] | null ()</h3>
```

Comment:
I'm afraid the final output does not look very good if you use this markup.

http://raw.githack.com/martinRenou/three.js/geometry_null_index/docs/index.html#api/en/core/BufferGeometry.getIndex

I think updating the TS files should be sufficient for now.

Reply:
Yes, I tried my best to make it look ok but that is the only thing I could do. I will undo the documentation changes then :)